package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@splunk/otel-web-dev-root",
+  "name": "@kloudmate/otel-web-dev-root",
```

Comment:
Rename the package `otel-js-web`?

Reply:
ok. But we won't be deploying this package. The one we will publish is a sub-package [here](https://github.com/kloudmate/otel-js-web/blob/update-config/packages/web/package.json#L2)

Reply:
okay